uroute/url.py

```diff
@@ -48,7 +48,7 @@ def _init_rules_data(self):
                 self.rules_data = json.load(rules_file)
             log.debug('URL cleaning rules loaded from %r', self.rules_path)
 
-    def clean_url(self, url):
+    def clean_url(self, url, loop = True):
```

Comment:
[PEP 8 recommends](https://www.python.org/dev/peps/pep-0008/#other-recommendations):

> Don't use spaces around the `=` sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter:

```suggestion
    def clean_url(self, url, loop=True):
```